src/general.scss

```diff
@@ -1,3 +1,7 @@
 @charset "utf-8";
 
 @import "bulma/bulma.sass";
+
+iframe {
+  display: none;
+}
```

Comment:
Delete this, it was useful while you was developing to avoid the react error iframe, but this shouldn't be in the production code